platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/sources/GeoJsonSource.java

```diff
@@ -133,8 +132,7 @@ public GeoJsonSource(String id, FeatureCollection features, GeoJsonOptions optio
    * @param features the GeoJSON {@link FeatureCollection}
    */
   public void setGeoJson(FeatureCollection features) {
```

Comment:
Is it the convention on Android to always have a Java wrapper for a native method, even when it's possible to expose the native method directly (e.g. `public native void setGeoJson(FeatureCollection features);`)?